pihole

```diff
@@ -16,6 +16,7 @@ readonly PI_HOLE_SCRIPT_DIR="/opt/pihole"
 # error due to modifying a readonly variable.
 setupVars="/etc/pihole/setupVars.conf"
 PI_HOLE_BIN_DIR="/usr/local/bin"
+readonly FTL_PID="/run/pihole-FTL.pid"
```

Comment:
Be specific that this is a file.
```suggestion
readonly FTL_PID_FILE="/run/pihole-FTL.pid"
```

Reply:
This has been borrowed from
https://github.com/pi-hole/pi-hole/blob/cbfb58f7a283c2a3e7aad95a834a0287175ccb24/advanced/Scripts/piholeDebug.sh#L133

I'm happy to change this.

Reply:
Pushed as a separate commit instead of amending to ease the review process. We can squash the entire thing in the end.